test/rules/object-literal-sort-keys/local-compare/test.ts.lint

```diff
@@ -0,0 +1,10 @@
+const a = {
+    familia: 1,
+    fútbol: 2
+}
+
+const b = {
+    fútbol: 2,
+    familia: 1
+    ~~~~~~~ [The key 'familia' is not sorted alphabetically]
```

Comment:
I copied your test to my checkout on master and it still seems to work without your code change. Can you write a test that would fail under the prior conditions but pass under the current ones please?

Reply:
- [x] Fixed.